README.rst

```diff
@@ -44,11 +44,54 @@ working) is via github or via IRC (#pimutils on Libera.Chat).
 
 Documentation
 -------------
-For khal's documentation have a look at the website_ or readthedocs_.
+For khal's documentation have a look at readthedocs_.
 
-.. _website: https://lostpackets.de/khal/
 .. _readthedocs: http://khal.readthedocs.org/
 
+Installation
+------------
+khal is packaged for most `operating systems`__ and should be installable with
+your standard package manager.
+
+.. __: https://repology.org/project/python:khal/versions
+
+For some exemplary OS you can find installation instructions below. Otherwise
+see the documentation_ for more information.
```

Comment:
One of these might read better. I'm leaning to "platforms" because of pip.

```suggestion
You can find installation instructions for some platforms below. Otherwise see
the documentation_ for more information.
```
```suggestion
You can find installation instructions for some operating systems below.
Otherwise see the documentation_ for more information.
```